analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.LearnConstraint.cs

```diff
@@ -138,9 +138,9 @@ public void Branching_BoolExpression_LearnsBoolConstraint()
     Tag(""Value"", value);
 }";
             var validator = SETestContext.CreateCS(code).Validator;
-            validator.ValidateTag("True", x => x.HasConstraint(BoolConstraint.True).Should().BeTrue());
+            validator.ValidateTag("True", x => x.HasConstraint(ObjectConstraint.NotNull).Should().BeTrue());
```

Comment:
Why this change? I'd expect it to has both, True and NotNull now.
As the test is about bool constraints, I'm fine asserting only `BoolConstraint.True` and ignoring `NotNull` here